medmodels/medrecord/schema.py

```diff
@@ -422,7 +422,7 @@ def __init__(
             )
 
     @classmethod
-    def _from_pyschema(cls, schema: PySchema) -> Schema:
+    def _from_py_schema(cls, schema: PySchema) -> Schema:
```

Comment:
same here, I kinda see Pyschema as one word, but this opinion is not super strong